Задачи 4модуль

TraktorT330 opened the pull request.

TraktorT330 committed:
1  задача модуль 4

Прошу пояснить, почему не проходит первый способ решения в скрипте index.js . Спасибо.

TraktorT330 committed:
1  задача модуль 4
Прошу пояснить, почему не проходит первый способ решения в скрипте index.js . Спасибо.
p/s в index.html добавлено <meta charset="UTF-8">

TraktorT330 committed:
Merge branch 'js-tasks-ru:master' into master

TraktorT330 committed:
1  задача модуль 4
Прошу пояснить, почему не проходит первый способ решения в скрипте index.js . Спасибо.
p/s в index.html добавлено <meta charset="UTF-8">

2 задача модуль 4 решение

TraktorT330 committed:
Merge branch 'js-tasks-ru:master' into master

TraktorT330 committed:
1  задача модуль 4
Прошу пояснить, почему не проходит первый способ решения в скрипте index.js . Спасибо.
p/s в index.html добавлено <meta charset="UTF-8">

2 задача модуль 4 решение
3 задача модуль 4 решение

jsru commented:
Добавляю преподавателя (@koalex) для код-ревью.

TraktorT330 committed:
Merge branch 'js-tasks-ru:master' into master

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

TraktorT330 committed:
1  задача модуль 4 решение
2 задача модуль 4 решение
3 задача модуль 4 решение

1 задача модуль 5 решение

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex

TraktorT330 committed:
Merge branch 'js-tasks-ru:master' into master

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

TraktorT330 committed:
1  задача модуль 4 решение
2 задача модуль 4 решение
3 задача модуль 4 решение

1 задача модуль 5 решение
2 задаче модуль 5 решение

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex

TraktorT330 committed:
Merge branch 'js-tasks-ru:master' into master

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

TraktorT330 committed:
1  задача модуль 4 решение
2 задача модуль 4 решение
3 задача модуль 4 решение

1 задача модуль 5 решение
2 задача модуль 5 решение
3 задача модуль 5 решение

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex

TraktorT330 committed:
1  задача модуль 4 решение
2 задача модуль 4 решение
3 задача модуль 4 решение

1 задача модуль 5 решение
2 задача модуль 5 решение
3 задача модуль 5 решение

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex

TraktorT330 committed:
1  задача модуль 4 решение
2 задача модуль 4 решение
3 задача модуль 4 решение

1 задача модуль 5 решение
2 задача модуль 5 решение
3 задача модуль 5 решение

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex
